invalid-exit.yml

```
@@ -29,3 +23,11 @@ steps:
         - CONTRACT_ADDRESS=$(env:PLASMA_ADDRESS)
         - CONTRACT_ABI=$(env:PLASMA_ABI)
     taskKey: execute
+    inputs:
+      method: challengeStandardExit
+      privateKey: $(env:PRIVATE_KEY)
```

PRIVATE_KEY should be something like `0x...` so this will be read as a number in yaml, We need to put the quotes.
```suggestion
      privateKey: "$(env:PRIVATE_KEY)"
```